.eslintrc

```diff
@@ -45,7 +45,8 @@
       "warn",
       {
         "endOfLine": "auto",
-        "singleQuote": true
+        "singleQuote": true,
+        "arrowParens": "avoid"
```

Comment:
Would be better to [configure prettier itself](https://prettier.io/docs/en/configuration.html) and let this plugin read those options. That way running the Prettier tool will use the same options as are being linted.

Though I recommend against this ESlint plugin anyway because there's no point in giving warnings about something that is automatic.

Reply:
Makes sense, removed the plugin and added a `.prettierrc` and it seems to work, just need to be sure to use prettier vscode extension, not just eslint.